nevergrad/optimization/optimizerlib.py

```diff
@@ -2055,6 +2078,114 @@ def _select_optimizer_cls(self) -> base.OptCls:
         return optimClass
 
 
+ProgD13 = ConfSplitOptimizer(num_optims=13, progressive=True,
+                             multivariate_optimizer=OptimisticDiscreteOnePlusOne).set_name("ProgD13", register=True)
+
+RecombiningOptimisticNoisyDiscreteOnePlusOne = ParametrizedOnePlusOne(crossover=True, mutation="discrete",
+                                                                      noise_handling="optimistic").set_name(
+    "RecombiningOptimisticNoisyDiscreteOnePlusOne", register=True)
```

Comment:
remove this, they don't need any registration. If you need them in the algo, just instantiate them there (without `set_name`)